docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,163 @@
+
+## Setting up the Bitbucket integration
+
+!!! note
+    -   The Bitbucket integration connects to a specific [workspace](https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-workspace/). You must be the **owner** of that Bitbucket workspace to proceed with the integration setup.
+    -   [See below](#bb-permissions) the permissions that Pulse requires from your Bitbucket account.  
+
+To set up the Bitbucket integration:
+
+1.  On Pulse, [expand **Integrations** and select **Bitbucket**](https://app.pulse.codacy.com/integrations/bitbucket){: target="_blank"}.
+
+1.  Click **Install Bitbucket App** and follow the instructions on the Bitbucket UI to install the app on your account.
+
+    ![Installing the Pulse Bitbucket App](images/bb-installing.png)
+
+1.  Wait until you get a confirmation that Pulse successfully connected the integration to Bitbucket.
+
+    ![Pulse Bitbucket integration set up successfully](images/bb-ok.png)
+
+    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+
+1.  Choose the workspace that you want to connect the integration. You will only be able to select a workspace that you are the owner.
+
+    ![Choosing the Bitbucket workspace](images/bb-workspace.png)
+
+1.  Click **Complete setup**.
+
+    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+
+## Automatic deployment detection using merged pull requests {: id="deployment-detection-merged-pr"}
+
+!!! note
+    <!--NOTE Limitation from Bitbucket integration V1-V2-->Pulse will use your merged pull requests to automatically detect your deployments. Other strategies to detect deployments will be available with a future version of Pulse.
+
+-   Pulse considers a deployment every merged pull request that **targets the default branch** of the repository.
+-   The deployment date is the timestamp when the corresponding pull request is merged.
+-   The set of changes in a deployment is the list of commits in the corresponding pull request. Pulse correctly tracks your changes even if you [squash or fast-forward the commits when merging the pull request](https://support.atlassian.com/bitbucket-cloud/docs/merge-a-pull-request/#Merge-strategies), since Pulse processes all the original commits before any changes to the Git history.
+-   Pulse associates all the Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding the user groups with less than two members. Pulse only takes the changes to Bitbucket user groups into account on pull requests merged after those changes.
+
+## Collected data
+
+The table below lists the data that the Bitbucket integration collects from your Bitbucket workspace, together with:
+
+-   The mapping between the data collected from Bitbucket and the [Pulse data model](https://ingestion.pulse.codacy.com/v1/api-docs#tocs_event)
+-   The metrics that Pulse calculates from the data to display on the dashboards
```

Comment:
If I understood correctly, I believe rewording the sentence may improve readability. Something like:
```suggestion
-   The metrics that Pulse displays on the dashboards, calculated using the collected data
```